CHANGELOG.md

```diff
@@ -2,609 +2,479 @@
 
```

Comment:
Please revert the changes in this file. It is automatically generated/updated

Reply:
done